src/theme-preval.js

```diff
@@ -305,6 +305,42 @@ const pagination = {
   }
 }
 
+const stateLabels = {
+  sizes: {
+    small: {
+      py: 1,
+      px: 2,
+      fontSize: 0
+    },
+    normal: {
+      py: 2,
+      px: '12px',
+      fontSize: 1
+    }
+  },
+
+  status: {
+    issueClosed: {
+      backgroundColor: 'red.6'
+    },
+    pullClosed: {
+      backgroundColor: 'red.5'
+    },
+    pullMerged: {
+      backgroundColor: 'purple.5'
+    },
+    issueOpened: {
+      backgroundColor: '#159739' // custom green
+    },
+    pullOpened: {
+      backgroundColor: '#2cbe4e' // This was generated by a sass function in Primer, so using a hex here
+    },
+    draft: {
+      backgroundColor: 'gray.5'
+    }
+  }
+}
```

Comment:
I believe we want to reference our theme values directly here as we do in the other theme objects. Doing so allows us to make the change I suggested above in the variant function also! I also made a suggested change here to make the open greens the same for pull requests and issues - I probably messed that up when I did the visual refresh changes.

```suggestion
const stateLabels = {
  sizes: {
    small: {
      padding: `${space[1]} ${space[2]}`,
      fontSize: fontSizes[0]
    },
    normal: {
      padding: `${space[2]} 12px`,
      fontSize: fontSizes[1]
    }
  },

  status: {
    issueClosed: {
      backgroundColor: red[5]
    },
    pullClosed: {
      backgroundColor: red[5]
    },
    pullMerged: {
      backgroundColor: purple[5]
    },
    issueOpened: {
      backgroundColor: '#159739' // custom green
    },
    pullOpened: {
      backgroundColor: '#159739' // custom green
    },
    draft: {
      backgroundColor: gray[5]
    }
  }
}
```

Reply:
Maybe I'm mistaken here, but the reason I went this direction is if we hardcode e.g. `purple[5]` into these variants, then if a user creates their own set of colors and merges in our theme with it, the variant definitions will still use the colors from *our* theme instead of looking up the user-supplied colors when it's time to apply the styles. I know we've used this pattern in a couple other places but I think we will end up needing to move away from it once we start work on color modes and want to use semantically meaningful primitive names.

Reply:
As I started writing up notes about the theming work we'll need to do for color modes, I realized that we'll already have to do a major audit of the theme anyway, so for now maybe it's best to just stick with the current pattern and address it all at once.